scripts/bootstrap.sh.in

```diff
@@ -171,6 +177,7 @@ main() {
     run "Stopping Salt minion service" stop_salt_minion_service
     run "Installing mandatory packages" install_packages "${PACKAGES[@]}"
     run "Configuring Salt minion to run in local mode" configure_salt_minion_local_mode
+    run "Ensure mountpoint is under /srv/scality" ensure_archives_mounted
```

Comment:
It's not really true, because if you provide a path in the bootstrap config it does not need to be mounted in `/srv/scality` to me it's more something like "Ensure MetalK8s archive is available for Salt-master and registry" (or maybe do not talk about salt and registry ...)

Reply:
True, I'll update this.

Reply:
Updated with `Ensure archive is available`